src/content/docs/reference/utils/VoltageController/motor-voltage-controller.md

```diff
@@ -0,0 +1,69 @@
+---
+title: 'Motor Voltage Controller'
+description: 'Motor voltage controller for the robot'
+---
+The MotorVoltageController is an adapter for an `frc::MotorController` to a VoltageController.
+
+**See also: [Voltage Controller](../../voltagecontroller)**

```

Comment:
[Voltage Controller](./index.md) Is what this should be

Reply:
what do you mean

Reply:
Sorry forgot abt markdown formatting working on GitHub I meant '[Voltage Controller](./index.md)' is what that should be